handler/deployment/deployment.go

```diff
@@ -25,9 +25,11 @@ import (
 )
 
 const (
-	deploymentURL = "%s/v1/job/%s/deployments"
-	planURL       = "%s/v1/job/%s/plan"
-	runURL        = "%s/v1/jobs"
+	infoURL		   	= "%s/v1/job/%s"
```

Comment:
Indentation.

Reply:
Strangely they were all level in my editor. Think I must have had tabs set up incorrectly.

I've fixed the indentation.

Reply:
👍